src/components/PageSwitch.tsx

```diff
@@ -1,9 +1,9 @@
 import { useCallback, useEffect, useMemo } from 'react'
-import { ChevronRight, ChevronLeft } from 'react-feather'
+import { ChevronRight, ChevronLeft } from 'lucide-react'
 import { useHistory } from 'react-router-dom'
 import styled from 'styled-components'
 import useQueryParams from '../hooks/useQueryParams'
-import { TextButton } from './Buttons'
+import { TextButton } from './Buttons/TextButton'
```

Comment:
I know this was not changed in this PR, but `Buttons/TextButton` should be a default export instead of a named export, similarly to the `ClipboardButton` and `QRCodeButton`, so that it can be imported like so:

```js
import TextButton from './Buttons/TextButton'
```

Reply:
It was changed in this PR actually. Nice catch.